### PR TITLE
Security fix and package update.

### PR DESCRIPTION
**Pull request by andrei-gavrilescu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,10 @@
         {
           "arch": "x64",
           "target": "AppImage"
+        },
+        {
+          "arch": "x64",
```
Comment (saghul) [inline package.json:44]: can you please drop this?

File: package.json [hunk 2]
```diff
@@ -70,7 +74,10 @@
     "webrtc",
     "xmpp",
     "electron",
-    "jitsi-meet"
+    "jitsi-meet",
+    "conference",
```
Comment (saghul) [inline package.json:78]: please drop this hunk too

Comment (andrei-gavrilescu) [reply package.json]: dropped

File: package.json (hunk 1 above)
Comment (andrei-gavrilescu) [reply package.json]: dropped

**Comment by awlx** [comment]
Just saying that it's not really cool, to implement stuff clearly coming from another PR without mentioning or refering to the original author. See "Version" Label ....

**Comment by ctheune** [comment]
Well. Really? This code was contributed by @awlx and provided in PR #225 .

Having those rewritten without proper author attribution is ... not very welcoming for people to contribute. I guess that you guys are really busy and I hate complaining, but, please. Posture!

Thanks @awlx for fixing those!

**Comment by saghul** [comment]
Please see the the commit: https://github.com/jitsi/jitsi-meet-electron/pull/227/commits/dc8483b13a9653bc129a319b5bd2f873b42323b5

**Comment by saghul** [comment]
We couldn't take the changes as-is so incorporated them used the well-know `Co-authored-by` note.

**Comment by ctheune** [comment]
Thanks for pointing that out. That really got lost in the amount of things. As I said: I really don't want to step on your toes, so thanks for pointing it out. It did cause a bit of confusion. Thanks. More communication is welcome to attract contributions, but again, I understand you are busy.

**Comment by saghul** [comment]
I see. Apologies for the confusion. We had to move quickly with a release, this was not known at the time the PR was opened. @awlx please accept my apologies and thanks for getting the ball rolling on the updates. 🙏

**Comment by awlx** [comment]
@saghul @andrei-gavrilescu 
Just as I pointed out in my original PR (#225), the build does now trigger Windows Defender Malware detection because of electron-builder 22.4.1.

Also Virustotal is triggered:
https://www.virustotal.com/gui/file/475984ba8326decd8f5c54dfd41ab8e03a79ccb93cc1ab1a16944c20d17325d4/detection

**Comment by saghul** [comment]
@awlx Damn, I missed that, sorry. No problem, we can do a beta3. Any chance you could PR the downgrade?

**Pull request merged**